app/tests/integration/nimbus/pages/review.py

```diff
@@ -1,8 +1,52 @@
 from nimbus.pages.base import Base
+from pypom import Region
 from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
 
 
 class ReviewPage(Base):
     """Experiment Review Page."""
 
+    _root_locator = (By.CSS_SELECTOR, "#PageRequestReview")
+    _approve_request_button_locator = (By.CSS_SELECTOR, "#approve-request-button")
+    _experiment_status_icon_locator = (
+        By.CSS_SELECTOR,
+        ".header-experiment-status .boarder-primary",
+    )
     _page_wait_locator = (By.CSS_SELECTOR, "#PageRequestReview")
```

Comment:
Should we just recycle `_root_locator` above (new L10), or does this serve a different purpose?